created a navbar and heroComponenet

iamluqmaan opened the pull request:
## List of changes
_Provide an overview_

## Screenshots
_Add before change and after change if it has been worked on previously_

## Steps on how to test it
_For example_
- Navigate to `/home` page
- Click on the about button
- It naviagtes to the `/about` page

iamluqmaan committed:
created a navbar and heroComponenet

iamluqmaan commented:
I'm not done working on the hover states of the navbar but that's what I've covered so far.

AOMuiz committed:
remove next-ui package

AOMuiz commented on package.json:
you shouldn't install any external package and if necessary consult first if its what we need.

AOMuiz commented on src/components/navbar.tsx:
```diff
@@ -0,0 +1,157 @@
+
+// import React from "react";
+// import {Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button, DropdownItem, DropdownTrigger, Dropdown, DropdownMenu} from "@nextui-org/react";
+// import {ChevronDown, Lock, Activity, Flash, Server, TagUser, Scale} from "./Icons.jsx";
```
make sure to remove any unused code when done.

AOMuiz commented on tailwind.config.ts:
```diff
@@ -47,6 +51,8 @@ const config: Config = {
       },
     },
+    darkMode: "class",
+    plugins: [nextui()],
```
remove code.

iamluqmaan committed:
Remove NextUIProvider and add Alexandria font

- Removed NextUIProvider from \_app.tsx
- Imported and added Alexandria font from Next.js font library
- Applied Alexandria font to the main component with the variable font-alexandria

iamluqmaan committed:
remove next_ui configurations

iamluqmaan committed:
responsiveness and dropdown when hover on navlinks

AOMuiz committed:
feat: convert hero to a component and move navbar to global
